data/json/items/armor/bandolier.json

```diff
@@ -105,6 +105,16 @@
     "pocket_data": [ { "ammo_restriction": { "410shot": 25, "shot": 25, "20x66mm": 25, "signal_flare": 25 }, "moves": 25 } ],
     "flags": [ "WATER_FRIENDLY", "WAIST", "OVERSIZE" ]
   },
+  {
+    "id": "torso_bandolier_grenade",
+    "type": "ARMOR",
+    "copy-from": "bandolier_shotgun",
+    "//": "It's very much similar to the torso shotgun bandolier I'm patterning this off of, so I'm keeping the data the same.",
+    "name": { "str": "torso grenade bandolier" },
+    "description": "A leather bandolier strapped around the torso for keeping 40mm grenades close to hand. For some reason, wearing this makes you feel Austrian.",
```

Comment:
```suggestion
    "description": "A leather bandolier strapped around the torso for keeping 40mm grenades close to hand.  For some reason, wearing this makes you feel Austrian.",
```

Reply:
You know something, I _keep on forgetting_ this style convention. 💢